### PR TITLE
Feature/add websocket functionallity

### PR DESCRIPTION
**Pull request by trinatafoya**
# Description
 This pull request is a feature that allows the `har-to-k6` converter to handle web-socket functionality. It uses the k6 library `import ws from 'k6/ws'`. These requests were formerly ignored in the original itteration of `har-to-k6`.
## Type of change
This version handles the websocket `send(...)` messages and adds a `setTimeout` in place of `sleep` for websockets, this is because k6 websocket callbacks are not made to work with the `sleep(...)` function.

### CODE REVIEW

File: src/render/websocket.js [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  if (messages.length > 1) {
+    let firstMessageTime = messages[0].time
+    let finalMessageTime = messages[messages.length - 1].time
+    let totalTimeMessagesTook = finalMessageTime - firstMessageTime
+    time = timeConnected - totalTimeMessagesTook
+  }
+  return time
+}
+
+function ws_send_messages(factor) {
```
Comment (NeedleInAJayStack) [inline src/render/websocket.js:53]: This function appears to be pretty GraphQL/PassiveLogic specific (referencing 44XX response codes as errors, using connection_init/connection_ack messages, etc). Is that the intent, or are these changes intended to be generic for all websocket traffic?

**Review by NeedleInAJayStack — COMMENTED** [review]
Looks really great! Just had a question on whether this is intended to be agnostic to the WebSocket subprotocol or tied to [graphql-ws](https://github.com/enisdenjo/graphql-ws/blob/master/PROTOCOL.md)

File: src/render/websocket.js (hunk 1 above)
Comment (trinatafoya) [reply src/render/websocket.js:53]: ah, yes. You are correct, I forgot to remove that, it was for us. Will fix.

File: src/aid.js [hunk 2]
```diff
@@ -53,8 +53,7 @@ function isBlacklistedHeader(headerName = '') {
   const [name] = headerName.split(';')
 
   return HEADERS_BLACKLIST.some(
-    (blacklistedHeader) =>
-      name.toLowerCase() === blacklistedHeader.toLowerCase()
+    blacklistedHeader => name.toLowerCase() === blacklistedHeader.toLowerCase()
   )
 }
```
Comment (trinatafoya) [inline src/aid.js:59]: Formatted in har-to-k6 with git hook.

File: src/aid.js [hunk 3]
```diff
@@ -35,8 +35,8 @@ function isNil(value) {
...
 
 function parseContentType(str = '') {
-  const [mimeType, ...rest] = str.split(';').map((s) => s.trim())
-  const params = objectFromEntries(rest.map((s) => s.split('=')))
+  const [mimeType, ...rest] = str.split(';').map(s => s.trim())
+  const params = objectFromEntries(rest.map(s => s.split('=')))
```
Comment (trinatafoya) [inline src/aid.js:39]: Formatted in har-to-k6 with git hook.

File: src/aid.js [hunk 4]
```diff
@@ -53,8 +53,7 @@ function isBlacklistedHeader(headerName = '') {
   const [name] = headerName.split(';')
 
   return HEADERS_BLACKLIST.some(
-    (blacklistedHeader) =>
-      name.toLowerCase() === blacklistedHeader.toLowerCase()
+    blacklistedHeader => name.toLowerCase() === blacklistedHeader.toLowerCase()
```
Comment (trinatafoya) [inline src/aid.js:56]: Formatted in har-to-k6 with git hook.

**Review by NeedleInAJayStack — APPROVED** [review]
Awesome! Nice job Katerina!
